core/src/main/java/gov/nasa/jpl/mudrod/utils/MatrixUtil.java

```diff
@@ -371,7 +386,15 @@ public Boolean call(Tuple2<String, Long> v1) throws Exception {
         String[] fields = t._1.split(",");
         String word = fields[0];
         int fieldsize = fields.length;
-        String[] numfields = Arrays.copyOfRange(fields, 1, fieldsize - 1);
+        int nStart = 1;
+        int nEnd = fieldsize - 1;
+        if (fieldsize < 2) {
+          System.out.println("***************output null***************");
```

Comment:
I'm talking about just the` System.out.println("***************output null***************");` statement.